GoogleChromeLabs/ps-analysis-tool
# Fix: Load settings when extension has been enabled

## Description
When in a regular profile where the extension is already installed, in this profile if the user disables and then enables the extension, the settings stored by service worker are destroyed. This PR aims to restore the extension settings once the extension is enabled.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch, in the terminal run `npm run build:all`.
- Load the extension in a new browser instance.
- Go to `example.com` then enable multi tab debugging and analyse 2 different tabs together.
- Now to go chrome://extensions page disable the extension and then enable it.
- Now you should still be able to see that multi tab debugging is still on.

## Additional Information:



## Screenshot/Screencast



---

## Checklist
